h2o-algos/src/main/java/hex/ensemble/StackedEnsembleModel.java

```diff
@@ -283,79 +282,72 @@ void checkAndInheritModelProperties() {
     boolean beenHere = false;
     boolean blending_mode = _parms._blending != null;
     boolean cv_required_on_base_model = !blending_mode;
+    boolean require_consistent_training_frames = !blending_mode && !_parms._is_cv_model;
     
     //following variables are collected from the 1st base model (should be identical across base models), i.e. when beenHere=false
     int basemodel_nfolds = -1;
     Parameters.FoldAssignmentScheme basemodel_fold_assignment = null;
     String basemodel_fold_column = null;
     long seed = -1;
     //end 1st model collected fields
-    
-    trainingFrameRows = _parms.train().numRows();
 
     for (Key<Model> k : _parms._base_models) {
       aModel = DKV.getGet(k);
       if (null == aModel) {
-        Log.warn("Failed to find base model; skipping: " + k);
+        Log.warn("Failed to find base model; skipping: "+k);
         continue;
       }
-      Log.debug("Checking properties for model " + k);
+      Log.debug("Checking properties for model "+k);
       if (!aModel.isSupervised()) {
-        throw new H2OIllegalArgumentException("Base model is not supervised: " + aModel._key.toString());
+        throw new H2OIllegalArgumentException("Base model is not supervised: "+aModel._key.toString());
       }
 
       if (beenHere) {
```

Comment:
@seb-h2o I know this is not you code, but I would rename `beenHere` as it is too generic.
```
 if(!isFirstBaseModel) {}
```
or 
```
 if(!isReferenceBaseModel) {}
```
would be clearer and we wont need some comments to explain `beenHere`. Ignore this if you don't feel like touching legacy code :)

Reply:
I agree, will rename